meta-lxatac-bsp/recipes-bsp/barebox/files/lxatac/env/init/20_serial_timeout.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# shellcheck disable=SC2154
```

Comment:
Instead of repeatedly disabling these in barebox scripts, perhaps there should be a `.shellcheckrc` in `meta-lxatac-bsp/recipes-bsp/barebox/files`?

Reply:
Sounds good, I have changed it in a separate commit.